packages/math/is_prime.pony

```diff
@@ -0,0 +1,80 @@
+primitive IsPrime[A: (Integer[A] val & Unsigned)]
+  """Primality test using 6k+-1 optimization."""
+
+  // Using a match table like below provides information to LLVM
+  // to allocate static globals rather than stack or heap allocation.
```

Comment:
Should this comment be on the _low_primate_table rather than the size function here? Or perhaps right before the table itself?

Reply:
I agree. It is why I had both methods without a newline as they were sort of acting as one method together. I moved the size method down below the table so the comment could be on top and be above the table method itself.